packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.tsx

```diff
@@ -1,63 +1,14 @@
-import React, { ReactElement } from "react";
+import React from "react";
 import classnames from "classnames";
 import ReactDOM from "react-dom";
-import { XOR } from "ts-xor";
 import styles from "./ComboboxContent.css";
-import { ComboboxSearch } from "./ComboboxSearch";
-import { ComboboxList } from "./ComboboxList";
+import { ComboboxSearch } from "./ComboboxContentSearch/ComboboxSearch";
+import { ComboboxList } from "./ComboboxContentList/ComboboxList";
 import { ComboboxContext } from "../../ComboboxProvider";
-import { ComboboxOption } from "../../Combobox.types";
+import { ComboboxContentProps, ComboboxOption } from "../../Combobox.types";
```

Comment:
I didn't do these on my own. these were suggestions/fixes from my IDE

it's curious to me that this resolves to a relative path with `../` whereas you can see some other components did `"@jobber/components/Combobox/Combobox.types";`

I kinda like the latter better. might change that myself and see if the autofixer hates that for some reason.